Source/FolioReaderCenter.swift

```diff
@@ -812,7 +812,9 @@ open class FolioReaderCenter: UIViewController, UICollectionViewDelegate, UIColl
             html = modifiedHtmlContent
         }
         
-        cell.loadHTMLString(html, baseURL: URL(fileURLWithPath: spine.resource.fullHref.deletingLastPathComponent))
+        let fileURL = URL(fileURLWithPath: spine.resource.fullHref)
```

Comment:
maybe add some comment here explaining the double: load fileUrl + loadHtml, also why removing 2 last path components is needed

Reply:
Done

Reply:
Thank you so much❤️